<!--
Copyright (C)  2016/2017 The University of Leeds and Rafael Papallas

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->
<launch>
    <node pkg="baxter_interface" type="joint_trajectory_action_server.py" name="trajectory_server"/>
    <include file="$(find baxter_moveit_config)/launch/baxter_grippers.launch" output="screen" args="-d $(find baxter_cashier_perception)/launch/rviz_config.rviz"/>
</launch>
